packages/core/src/components/TextField/Mask.test.jsx

```diff
@@ -298,6 +298,13 @@ describe('unmask', () => {
     expect(unmask(null)).toBeNull();
   });
 
+  it('returns empty string when there are no numeric characters in the value', () => {
+    expect(unmask('banana', 'currency')).toBe('');
+    expect(unmask('banana', 'zip')).toBe('');
+    expect(unmask('banana', 'ssn')).toBe('');
+    expect(unmask('banana', 'phone')).toBe('');
+  });
+
```

Comment:
What do you think about adding some tests to check for proper rendering of both non-numeric, and numeric with extra junk characters, just to ensure there's no regression? Up further in this file in the `Currency` describe block.